src/Services/Character/Character.API/Exceptions/CharacterNotFoundException.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Common.Exceptions;
+
+namespace Character.API.Exceptions
+{
+    public class CharacterNotFoundException(Guid Id) : NotFoundException("Character", Id)
```

Comment:
Could we do NotFoundException(nameof(Entities.Character), Id) ?
The less magic strings the better